<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentForm</class>
 <widget class="QWidget" name="DocumentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="doc_twidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>25</y>
     <width>496</width>
     <height>151</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Doc No.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Creation Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Author</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>File Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Doc Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Note</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Documents</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_doc_button">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>25</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add Document</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/plus.png</normaloff>:/wnt/plus.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_doc_button">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>55</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>View/Edit Document Details</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/edit.png</normaloff>:/wnt/edit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="view_doc_button">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>115</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>View Document</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/eye.png</normaloff>:/wnt/eye.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_doc_button">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>85</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete Document</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/minus.png</normaloff>:/wnt/minus.png</iconset>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>doc_twidget</tabstop>
  <tabstop>add_doc_button</tabstop>
  <tabstop>edit_doc_button</tabstop>
  <tabstop>delete_doc_button</tabstop>
  <tabstop>view_doc_button</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
